app/instances.go

```diff
@@ -189,7 +189,7 @@ func (a *App) createTriggers(tConfigs []*trigger.Config, runner action.Runner) (
 						if ok {
 							s, err := schema.FindOrCreate(ref)
 							if err != nil {
-								return nil, err
+								return nil, fmt.Errorf("error finding or creating output schema [%s] in trigger [%s]'s handler [%s]: %s", ref, tConfig.Id, hConfig.Name, err.Error())
```

Comment:
fmt.Errorf("unable to find or create output schema [%s] for trigger [%s]'s handler [%s]: %s"